conf/ledgersmb-apache-2.4.template

```diff
@@ -0,0 +1,60 @@
+# This file must be edited to substitute the current path with WORKING_DIR
+
+Alias /lsmbcss /var/lib/master/css/
+
+<VirtualHost *:80>
+
+  ServerName [SERVERNAME]:80
+  DocumentRoot WORKING_DIR
+
+  <Proxy *>
+    Order Allow,Deny
+    Allow from All
+  </Proxy>
+
+  LogLevel Warn
+
+  <FilesMatch ...>
+    SetHandler  "proxy:pl://localhost:5001"
+  </FilesMatch>
+
+  ProxyRequests     Off
+  ProxyPreserveHost On
+  ProxyVia          On
+  ProxyPass        / http://localhost:5001/
+  ProxyPassReverse / http://localhost:5001/
+
+</VirtualHost>
+
+# The rest of this file just tightens up security.
```

Comment:
With the unconditional reverse-proxy above, there's really no need to tighten up access: you haven't opened up the access to the WORKING_DIR directory to begin with...